cedar-policy-validator/src/cedar_schema/parser.rs

```diff
@@ -83,33 +83,33 @@ lazy_static::lazy_static! {
     static ref TYPE_PARSER: grammar::TypeParser = grammar::TypeParser::new();
 }
 
-/// Parse errors for parsing a human-syntax schema
+/// Parse errors for parsing a schema in the Cedar syntax
 #[derive(Debug, Diagnostic, Error)]
-pub enum HumanSyntaxParseErrors {
-    /// Parse error for the human syntax
+pub enum CedarSyntaxParseErrors {
```

Comment:
I think the name `CedarSchemaParseErrors` would better convey that this error is schema specific

Reply:
Done.